src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    public void getRandomBall() {

```

Comment:
```suggestion
    public Ball getRandomBall() {

```
краще повертати об'єкт класу і вже в залежності від контексту виводити його в консоль або робити щось інше з ним